x/stakeibc/keeper/icacallbacks_detokenize.go

```diff
@@ -61,8 +63,21 @@ func (k Keeper) DetokenizeCallback(ctx sdk.Context, packet channeltypes.Packet,
 	// If the ICA succeeded, remove the token deposit
 	k.RecordsKeeper.RemoveLSMTokenDeposit(ctx, deposit.ChainId, deposit.Denom)
 
+	// Determine the actual number of tokens that were turned to native stake
+	// (this can be slightly different than the amount initiated in the redeem tokens tx
+	// due a precision error in the SDK)
+	if len(ackResponse.MsgResponses) != 1 {
+		return fmt.Errorf("Invalid number of messages (%d) in detokenize response: %v",
```

Comment:
Do we ever hit this?

Reply:
We shouldn't cause only one msg is submitted at a time
https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/keeper/lsm.go#L296

(Also confirmed in dockernet there is only 1 tx)

Reply:
makes sense, resolved